pandas/tests/indexing/interval/test_interval_new.py

```diff
@@ -128,6 +128,7 @@ def test_loc_with_slices(self):
         with pytest.raises(NotImplementedError, match=msg):
             s[Interval(3, 4, closed="left") :]
 
+        # FIXME: dont leave commented-out
```

Comment:
I think you can just remove it, or turn it in a test. It's the exact same error and message as the tests just above with other non-existing intervals.